.github/workflows/pipeline_approval.yml

```
@@ -75,7 +75,7 @@ jobs:
               const approvers = [...approvalsSet];
               const rejecters = [...rejectionsSet];
               const awaiting = awaitingArr;
-              let body = `## Pipeline approval status: ${status}\n\nPipeline has approvals from ${approvalsSet.size}/${quorum} required @core-team quorum.\n\n`;
+              let body = `## Pipeline approval status: ${status}\n\nPipeline has approvals from ${approvalsSet.size}/${quorum} required @core-team and @maintainers-team quorum.\n\n`;
```

```suggestion
              let body = `## Pipeline approval status: ${status}\n\nPipeline has approvals from ${approvalsSet.size}/${quorum} required @nf-core/core and @nf-core/maintainers quorum.\n\n`;
```
`@nf-core/core` and `@nf-core/maintainers` are the actual teams

---

better than pinging someone not related

---

No but I mean every person is pinged by name already